test/utilsWithFlattening.spec.js

```diff
@@ -17,7 +17,7 @@ describe("Utils", () => {
           "id",
         );
 
-        const result = await typesenseDocumentFromSnapshot(documentSnapshot);
+        const result = await typesenseDocumentFromSnapshot(documentSnapshot, {});
```

Comment:
What I was referencing in my earlier comment

Reply:
will do